src/store/consts/uiUtils.ts

```diff
@@ -58,3 +59,14 @@ export const tickerToAddress = (ticker: string): string => {
 export const addressToTicker = (address: string): string => {
   return reversedAddressTickerMap[address] || address
 }
+
+export const initialXtoY = (tokenXAddress?: string, tokenYAddress?: string) => {
+  if (!tokenXAddress || !tokenYAddress) {
+    return true
+  }
+
+  const isTokeXStablecoin = STABLECOIN_ADDRESSES.some(address => address === tokenXAddress)
```

Comment:
You can use a more precise function here: `includes` instead of `some`.

```Typescript
const isTokeXStablecoin = STABLECOIN_ADDRESSES.includes(tokenXAddress);
```